deb/build_deb.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+cd ..
+make dist
+cd deb
+cp ../fapolicyd-*.tar.gz .
+
+tar zxvf fapolicyd-*.tar.gz
+cd fapolicyd-*/
+
+# Ugly work around for INSTALL.tmp
+# Need to figure out proper fix.
+mv INSTALL INSTALL.tmp
```

Comment:
The `INSTALL.tmp` file is not in the dist tar ball. This causes the build to fail. This is a hacky workaround and would love a proper solution.